styles/Events.module.scss

```diff
@@ -44,6 +44,15 @@
   grid-template-rows: 1fr 1fr 1rem;
   grid-template-columns: 200px 1fr;
   text-decoration: none;
+  transition: .3s ease;
```

Comment:
Please add the elements that expect to be changed https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property